### ISSUE TITLE
JSON circular error in v6.0.0

### ISSUE THREAD

**Issue by joehand** [sethvincent/appa#3]
I keep getting a `TypeError: Converting circular structure to JSON` error in v6.0.0 of Appa (full stack trace below). Downgrading to 5.0.0 fixes error.

```
  return JSON.stringify(view)
              ^

TypeError: Converting circular structure to JSON
    at Object.stringify (native)
    at Object.typemap.json (/Users/joe/node_modules/archiver-rest/node_modules/response/index.js:161:15)
    at Function._response [as json] (/Users/joe/node_modules/archiver-rest/node_modules/response/index.js:168:51)
    at Function.send (/Users/joe/node_modules/archiver-rest/node_modules/appa/send.js:9:19)
    at /Users/joe/node_modules/archiver-rest/server.js:14:9
    at ArchiverRest.add (/Users/joe/node_modules/archiver-rest/index.js:20:10)
    at /Users/joe/node_modules/archiver-rest/server.js:12:7
    at /Users/joe/node_modules/archiver-rest/node_modules/appa/index.js:70:16
    at emit (/Users/joe/node_modules/archiver-rest/node_modules/wayfarer/index.js:51:22)
    at Server.app (/Users/joe/node_modules/archiver-rest/node_modules/appa/index.js:45:12)
```

**Comment by sethvincent** [comment]
I think this happens because I changed the signature of the `send` function in v6 and I also forgot to update the change log 😦.

In v5 of appa it was this:

```js
send(res, { obj: 'whatevs' })
```

in v6 it's this:

```js
send({ obj: 'whatevs' }).pipe(res)
```

I did this so it feels a little less magic, and makes it so if you want to use `pump` and/or do any transforms for whatever reason, that's possible.

I'll update the docs with better notice / examples. But also let me know if it seems like this isn't actually the reason.
